### ISSUE TITLE
Sum of Density Map

### ISSUE THREAD

**Issue by guanning03** [cvlab-stonybrook/LearningToCountEverything#49]
I ran the demo and found that the total sum of the oranges are 29.16, which is lower than the total number of the image. However, the generated density map clearly shows the location of every orange. So why the sum of oranges is inaccurate? Is this phenomenon normal or I didn't correctly ran the demo?
![image](https://github.com/user-attachments/assets/5a702853-7008-4652-bfe1-7f5f1a2417c8)

**Comment by huangjindiao** [comment]
Yes, I also found that the test results and the actual number of pictures do not match.
